inc/Engine/Optimization/DelayJS/HTML.php

```diff
@@ -230,6 +230,11 @@ public function replace_scripts( $matches ): string {
 			}
 		}
 
+		// Checks if script has unnecessary white space and treat as included js script.
+		if ( '' === trim( $matches['content'] ) ) {
```

Comment:
```suggestion
		if ( empty( trim( $matches['content'] ) ) ) {
```

You still can use empty function instead of strict condition